source/documentation/standards/metrics.html.md.erb

```diff
@@ -0,0 +1,59 @@
+---
+title: Metrics
+last_reviewed_on: 2021-03-15
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## User needs
+
+Users should be able to label metric data so that it is compatible with
+UK Government PaaS Exporter metrics and common across applications.
```

Comment:
Should we list the metrics reserved by paas?

Reply:
No, but I will put a link to https://github.com/alphagov/paas-prometheus-exporter